lightning-tag/basic-implementation.md

```diff
@@ -0,0 +1,42 @@
+# Base Lightning Tag Implementation 
+
+!> This document provides information on how to implement Lightning Tag in its basic form. Throughout this page, there will be links to documentation detailing additional features and use-cases that can be deployed
+
+Setup of the Lotame Lightning Tag javascript on your website involves an input object that gets passed to an initialization function that prepares a window namespace on your page for the javascript tag itself. 
+
+Lotame recommends loading the Lightning Tag directly on your page, in the <head> section. While not always possible, Lotame recommends against loading through a tag manager such as Google Tag Manager (GTM)
+
+The basic setup looks like the below:
+
+```javascript 
```

Comment:
does this need to include the js file itself in a script src tag?  in itself this doesn't need that code, but for this code to do anything it does?

Reply:
or mention that this is just the configuration and you still need to load the actual code?

Reply:
We should include the script tag as well as the header links for preconnect stuff.